lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -1913,7 +1913,7 @@ namespace Js
         for (int i = 0; i < length; i += 1)
         {
             Var value;
-            DynamicObject *obj = DynamicObject::FromVar(pScope->GetItem(i));
+            RecyclableObject *obj = RecyclableObject::FromVar(pScope->GetItem(i));
```

Comment:
Presumably this is only called from within eval, and so all the contents of the FrameDisplay are objects.